src/main/scala/cromwell/engine/backend/io/filesystem/gcs/GcsFileSystem.scala

```diff
@@ -6,21 +6,26 @@ import java.nio.file.attribute.UserPrincipalLookupService
 import java.nio.file.spi.FileSystemProvider
 import java.util.{Collections, Set => JSet}
 
-import cromwell.engine.PathString
+import scala.language.postfixOps
 
-import scala.concurrent.ExecutionContext
-import scala.util.Try
+case class NotAGcsPathException(path: String) extends IllegalArgumentException(s"$path is not a valid GCS path.")
 
 object GcsFileSystem {
-  import PathString._
-  def apply(interface: Try[GoogleCloudStorage], root: String)(implicit executionContext: ExecutionContext) = {
-    if (root.isGcsUrl) new GcsFileSystem(GcsFileSystemProvider(interface), root)
-    else throw new IllegalArgumentException(s"$root is not an absolute GCS path")
+  def apply(provider: GcsFileSystemProvider, root: String) = {
+    new GcsFileSystem(provider , root)
```

Comment:
Example of an extra space. In general, this file could use a "Reformat Code ⌥⌘L"